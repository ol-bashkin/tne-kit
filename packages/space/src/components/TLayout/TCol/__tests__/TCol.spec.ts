import { mount } from "@vue/test-utils";
import WCol from "../TCol";

describe("TCol.ts", () => {
  test("should have an t-col class", () => {
    const wrapper = mount(WCol, {
      propsData: {
        col: "2",
      },
    });
    expect(wrapper.classes()).toContain("t-col");
    expect(wrapper.html()).toMatchSnapshot();
  });
});
