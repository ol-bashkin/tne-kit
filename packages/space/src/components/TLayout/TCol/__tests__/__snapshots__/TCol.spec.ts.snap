// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TCol.ts should have an t-col class 1`] = `<div class="t-col col col-2"></div>`;
